src/functions/user_screen.py

```diff
@@ -93,13 +97,22 @@ def enable_sudo(self, widget, switch_state):
 
     def verify_password(self, widget):
         if self.password_entry.get_text() == self.password_confirmation.get_text():
```

Comment:
```suggestion
        if self.password_entry.get_text() == self.password_confirmation.get_text() and self.password_entry.get_text() != "" and self.password_confirmation.get_text() != "":
```

Reply:
id say check if the password fields are null if you don't someone can type password on both fields then remove the password and it still will let you go through

Reply:
this might not be the best way to do it tho

Reply:
Good catch, I simplified the check to just `and self.password_entry.get_text().strip()` since that would also cover if the password entry is only whitespace